### DISCUSSION TITLE
Vercel 重新部署失败

### DISCUSSION THREAD

**Discussion by Godoword** [lobehub/lobehub#1698] [Self Hosting]
### 💻 系统环境

Windows

### 📦 部署环境

Vercel / Zeabur / Sealos

### 🌐 浏览器

Chrome

### 🐛 问题描述

按照“自动同步更新”的步骤操作，进行重新部署的时候提示The provided GitHub repository can't be found.或者The provided GitHub repository does not contain the requested branch or commit reference. Please ensure the repository is not empty.

我不确定是我的操作问题还是vercel的问题，以下是我的步骤：

1、在lobe-chat的README页面，点击“使用 Vercel 部署”，授权，按提示一步步进行，部署完成，自定义的域名可以正常访问，导入数据以及正常使用正常。
2、按文档提示，在Github删除了由vercel创建的仓库。
3、Fork本项目，Actions 页面启用 Workflows，并启动 Upstream Sync Action。
4、在Vercel点击重新部署，出现我在开头提到的错误提示。
5、查看vercel的settings/git部分，有一个Error: Project Link not found的提示，按钮变成Reconnect

尝试了几次仍是如此，不过之前部署的域名访问使用正常。

### 🚦 期望结果

希望能解答疑惑，并确保我自己的部署能自动同步更新。谢谢。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
你可以点击reconnect 重新绑定到你的新仓库

[a comment by an automated account is not shown]

**Comment by Godoword** [comment]
> 你可以点击reconnect 重新绑定到你的新仓库

昨晚Reconnect并重新绑定到新仓库尝试过了，忘了说明这一步骤，重新部署仍然得到之前的错误提示。如图：

![canvas](https://github.com/lobehub/lobe-chat/assets/5213106/b4fccb32-a19d-46ed-adfb-7fee76318112)
![canvas1](https://github.com/lobehub/lobe-chat/assets/5213106/5fe39d1b-193a-424d-8651-6906b088ca44)
![canvas2](https://github.com/lobehub/lobe-chat/assets/5213106/8dea7f6e-ee74-4959-aaf0-f5d8398eaa86)
![4](https://github.com/lobehub/lobe-chat/assets/5213106/eb89de81-ec64-44c0-94c4-8498a8e80f7a)

**Comment by JimXiongGM** [comment]
The same question.
